.travis.yml

```diff
@@ -21,48 +28,39 @@ jobs:
         - NODE=$(which node) make lint lint-py
 
     - name: "Compile V8"
-      language: cpp
       cache: ccache
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
             - g++-6
-      install:
-        - CC='ccache gcc-6' CXX='ccache g++-6' ./configure
+      install: *ccache-setup-steps
       script:
-        - CC='ccache gcc-6' CXX='ccache g++-6' make -j2 -C out V=1 v8
+        - ./configure
+        - make -j2 -C out V=1 v8
 
     - name: "Compile Node.js"
-      language: cpp
       cache: ccache
       addons:
         apt:
           sources:
             - ubuntu-toolchain-r-test
           packages:
             - g++-6
-      install:
-        - CC='ccache gcc-6' CXX='ccache g++-6' ./configure
+      install: *ccache-setup-steps
       script:
-        - CC='ccache gcc-6' CXX='ccache g++-6' make -j2 V=1
+        - ./configure
+        - make -j2 V=1
+        - make -j1 V=1 build-addons build-js-native-api-tests build-node-api-tests
+        - python tools/test.py -j 2 -p dots --mode=release --flaky-tests=dontcare addons js-native-api node-api
+        - cp out/Release/node /home/travis/.ccache
 
     - stage: "Tests"
       name: "Test Suite"
-      language: cpp
       cache: ccache
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - g++-6
       install:
-        - export CC='ccache gcc-6' CXX='ccache g++-6' JOBS=2
-        - ./configure
-        # We already have a compile log in the above job
-        - make -j2 > /dev/null
-        - make -j1 build-addons build-js-native-api-tests build-node-api-tests > /dev/null
+        - mkdir -p out/Release
+        - cp /home/travis/.ccache/node out/Release/node
       script:
-        - JOBS=2 FLAKY_TESTS=skip make -s -j1 V= test-ci | grep -F -e "---" -e "..." -v
+        - python tools/test.py -j 2 -p dots --mode=release --flaky-tests=dontcare parallel
```

Comment:
Maybe run with `--report` to get a count/summary of the tests?

This PR isn't running all test suites (e.g. sequential, report, ...).

Reply:
`--report` sounds good.

Running only `parallel` was in order to make a point ;)
But as this work progressed it seems like we could simply use this trick to run two parallel Travis jobs, and get back the test coverage.

Reply:
Done